content/how-to/dedicated-hosting/add-cpu-to-server-or-hypervisor/index.md

```diff
@@ -15,29 +15,36 @@ This article explains how to create a ticket to add a CPU to a server or hypervi
 
 ### Create a ticket in the MyRackspace portal
 
-1. Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
+**Step 1.** Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
    your username and password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="Enter the login credentials with Username and Password" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor4.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+**Step 2.** In the top navigation bar, choose **Support** > **Support Center**.
 
-4. Select the **Subject** field.
+   <img width="941" alt="Select Support Center from top navigation bar" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor5.png">
 
-5. From the drop-down menu, Select **Add CPU to Server or Hypervisor**.
+**Step 3.** Under **Common Request**, click **See All Templates**. The **See All Templates** displays list of all templates.
 
-   {{<image src="hypervisor1.png" alt="" title="">}}
+   <img width="958" alt="Displays the list of available templates" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor6.png">
 
-6. In the **Ticket Details** section, select a device from the drop-down menu and
+**Step 4.** From the **Common Request** menu, select **Add CPU to Server or Hypervisor**.
+
+  <img width="249" alt="Select add CPU to server or hypervisor from common request menu" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor7.png">
+
+**Step 5.** In the **Issue Details** section, select a device from the drop-down menu and
    enter the number of CPUs that you want to add.
 
-   {{<image src="hypervisor2.png" alt="" title="">}}
+  <img width="524" alt="Select the required device from drop-down menu" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor8.png">
+
+**Step 6.** In the **Preferred Maintenance Schedule** section, you can choose to have the
+   work done at any time by selecting the **Anytime** checkbox. Fill out the required fields such as contact name, phone number and select the appropriate radio button as shown in following image. 
+
+   <img width="275" alt="Checkbox Anytime is tick so that you don't have to decide day and time" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">
```

Comment:
```suggestion
   <img width="275" alt="Checkbox Anytime is selected so that you don't have to decide day and time" src="/support/how-to/add-cpu-to-server-or-hypervisor/hypervisor9.png">
```